Document/0x07b-Testing-Network-Communication.md

```diff
@@ -227,3 +227,87 @@ Ensure that critical operations require at least one additional channel to confi
 * [1] The Mobile Application Security Verification Standard - https://github.com/OWASP/owasp-masvs/blob/master/Document/0x03-Using_the_MASVS.md
 * [2] Infobip 2FA library - https://2-fa.github.io/libraries/android-library.html
 * [3] Google Authenticator for Android - https://github.com/google/google-authenticator-android
+
+
+### Testing SSL/TLS Pinning
+
+#### Overview
+
+Certificate pinning allows to hard-code the certificate or parts of it into the app that is known to be used by the server<sup>[1]</sup>. This technique is used to reduce the threat of a rogue CA and CA compromise. Pinning the server’s certificate takes the CA out of the game. Mobile apps that implement certificate pinning only can connect to a limited numbers of servers, as a small list of trusted CAs or server certificates are hard-coded in the application.
```

Comment:
> Certificate pinning allows to hard-code the certificate or parts of it into the app that is known to be used by the server

I would place "that is known to be used by the server" closer to "certificate". Currently it looks like the app is used by the server.

> This technique is used to reduce the threat of a rogue CA and CA compromise.

I think it should be "or". Also, maybe you can swap the two so it is clear that you don't mean a "rogue CA compromise". So it should be something like "of a CA compromise or rogue CA."